resources/netregexes.ts

```diff
@@ -522,4 +548,35 @@ export default class NetRegexes {
       6: { field: 'toggle' },
     });
   }
+
+  static parse<T extends ParamsAllTypes>(regexpString: RegExp | string): NetRegex<ParamsAll[T]> {
+    const kCactbotCategories = {
```

Comment:
Is there a way to not duplicate this `(regexpString: RegExp | string) => RegExp` common code?